query/sources/common/src/main/java/org/codice/ddf/admin/sources/fields/type/OpensearchSourceConfigurationField.java

```diff
@@ -29,7 +31,7 @@ public OpensearchSourceConfigurationField() {
 
     @Override
     public List<Field> getFields() {
-        // TODO: tbatie - 2/28/17 - Add additional fields once implemented
-        return super.getFields();
+        return new ImmutableList.Builder<Field>().addAll(super.getFields())
```

Comment:
Added the TODO back but left this method here to make the TODO's purpose more clear.